### PR TITLE
[docs] consolidate the coordination nodes content

### PR DESCRIPTION
**Pull request by blinkov**
### Changelog entry 

...

### Changelog category 

* Documentation (changelog entry is not required)

### Additional information

Supersedes #7657

### CODE REVIEW

**Review by snaury — DISMISSED** [review]

**Review by alex268 — DISMISSED** [review]

File: ydb/docs/en/core/reference/ydb-sdk/coordination.md [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+
+- Go
+
+    {% list tabs group=lang %}
+
+    - Go
+
+        ```go
+        err := db.Coordination().CreateNode(
+            ctx,
```
Comment (asmyasnikov) [inline ydb/docs/en/core/reference/ydb-sdk/coordination.md:19]:
```suggestion
        err := db.Coordination().CreateNode(ctx,
```

File: ydb/docs/ru/core/reference/ydb-sdk/coordination.md [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+
+Узлы координации создаются в базах данных {{ ydb-short-name }} в том же пространстве имён, что и другие объекты схемы, такие как [таблицы](../../concepts/datamodel/table.md) и [топики](../../concepts/topic.md).
+
+{% list tabs %}
+
+- Go
+
+    ```go
+    err := db.Coordination().CreateNode(
+        ctx,
```
Comment (asmyasnikov) [inline ydb/docs/ru/core/reference/ydb-sdk/coordination.md:15]:
```suggestion
    err := db.Coordination().CreateNode(ctx,
```

File: ydb/docs/ru/core/reference/ydb-sdk/coordination.md [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
+
+Для начала работы клиент должен установить сессию, в рамках которой он будет осуществлять все операции с узлом координации.
+
+{% list tabs %}
+
+- Go
+
+    ```go
+    session, err := db.Coordination().CreateSession(
+        ctx,
```
Comment (asmyasnikov) [inline ydb/docs/ru/core/reference/ydb-sdk/coordination.md:59]:
```suggestion
    session, err := db.Coordination().CreateSession(ctx,
```

File: ydb/docs/ru/core/reference/ydb-sdk/coordination.md [hunk 4]
```diff
@@ -0,0 +1,306 @@
...
+
+Чтобы захватить семафор, клиент должен вызвать метод `AcquireSemaphore` и дождаться получения специального объекта `Lease`. Этот объект представляет собой подтверждение о том, что значение семафора было успешно увеличено и может считаться таковым до явного отпускания такого семафора или завершения сессии, в которой такое подтверждение было получено.
+
+{% list tabs %}
+
+- Go
+
+    ```go
+    lease, err := session.AcquireSemaphore(ctx,
+        "mySemaphore",  // semaphore name
```
Comment (asmyasnikov) [inline ydb/docs/ru/core/reference/ydb-sdk/coordination.md:158]:
```suggestion
        "my-semaphore",  // semaphore name
```

File: ydb/docs/ru/core/reference/ydb-sdk/coordination.md [hunk 5]
```diff
@@ -0,0 +1,306 @@
...
+
+Взятое значение захваченного семафора можно снизить (но не увеличить), вновь вызвав для него метод `AcquireSemaphore` с меньшим значением.
+
+### Обновление данных семафора {#update-semaphore}
+
+С помощью метода `UpdateSemaphore` можно обновить (заменить) данные семафора, которые были привязаны при его создании.
+
+{% list tabs %}
+
+- C++
```
Comment (asmyasnikov) [inline ydb/docs/ru/core/reference/ydb-sdk/coordination.md:205]:
```suggestion
- Go

    ```go
    err := session.UpdateSemaphore(
        "my-semaphore",                                                          // semaphore name
        options.WithUpdateData([]byte("updated-data")),   // new semaphore data
    )
    ```

- C++
```

File: ydb/docs/ru/core/reference/ydb-sdk/coordination.md [hunk 6]
```diff
@@ -0,0 +1,306 @@
...
+
+{% endlist %}
+
+Этот вызов не требует захвата семафора и не приводит к нему. Если требуется, чтобы данные обновлял только один конкретный клиент, то это необходимо явным образом обеспечить, например, захватив семафор, обновив данные и отпустив семафор обратно.
+
+### Получение данных семафора {#describe-semaphore}
+
+{% list tabs %}
+
+- C++
```
Comment (asmyasnikov) [inline ydb/docs/ru/core/reference/ydb-sdk/coordination.md:225]:
```suggestion
- Go

    ```go
    description, err := session.DescribeSemaphore(
        "my-semaphore"                                // semaphore name
        options.WithDescribeOwners(true), // to get list of owners
        options.WithDescribeWaiters(true), // to get list of waiters
    )
    ```

- C++
```

File: ydb/docs/en/core/reference/ydb-sdk/coordination.md [hunk 7]
```diff
@@ -0,0 +1,314 @@
...
+
+{% endlist %}
+
+This call doesn't require acquiring the semaphore and doesn't lead to it. If you need only one specific client to update the data, this must be explicitly ensured, for example, by acquiring the semaphore, updating the data, and releasing the semaphore back.
+
+### Getting semaphore data {#describe-semaphore}
+
+{% list tabs %}
+
+- C++
```
Comment (asmyasnikov) [inline ydb/docs/en/core/reference/ydb-sdk/coordination.md:233]:
```suggestion
- Go

    ```go
    description, err := session.DescribeSemaphore(
        "my-semaphore"                                // semaphore name
        options.WithDescribeOwners(true), // to get list of owners
        options.WithDescribeWaiters(true), // to get list of waiters
    )
    ```

- C++
```

File: ydb/docs/en/core/reference/ydb-sdk/coordination.md [hunk 8]
```diff
@@ -0,0 +1,314 @@
...
+
+To start working with coordination nodes, a client must establish a session within which it will perform all operations with the coordination node.
+
+{% list tabs %}
+
+- Go
+
+    ```go
+    session, err := db.Coordination().CreateSession(
+        ctx,
```
Comment (asmyasnikov) [inline ydb/docs/en/core/reference/ydb-sdk/coordination.md:65]:
```suggestion
    session, err := db.Coordination().CreateSession(ctx,
```

File: ydb/docs/en/core/reference/ydb-sdk/coordination.md [hunk 9]
```diff
@@ -0,0 +1,314 @@
...
+
+To acquire a semaphore, the client must call the `AcquireSemaphore` method and wait for a special `Lease` object. This object represents confirmation that the semaphore value was successfully increased and can be considered as such until explicit release of such semaphore or termination of the session in which such confirmation was received.
+
+{% list tabs %}
+
+- Go
+
+    ```go
+    lease, err := session.AcquireSemaphore(ctx,
+        "mySemaphore",  // semaphore name
```
Comment (asmyasnikov) [inline ydb/docs/en/core/reference/ydb-sdk/coordination.md:164]:
```suggestion
        "my-semaphore",  // semaphore name
```

File: ydb/docs/en/core/reference/ydb-sdk/coordination.md [hunk 10]
```diff
@@ -0,0 +1,314 @@
...
+
+The taken value of an acquired semaphore can be decreased (but not increased) by calling the `AcquireSemaphore` method again with a smaller value.
+
+### Updating semaphore data {#update-semaphore}
+
+Using the `UpdateSemaphore` method, you can update (replace) the semaphore data that was attached during its creation.
+
+{% list tabs %}
+
+- C++
```
Comment (asmyasnikov) [inline ydb/docs/en/core/reference/ydb-sdk/coordination.md:213]:
```suggestion
- Go
    ```go
    err := session.UpdateSemaphore(
        "my-semaphore",                                                          // semaphore name
        options.WithUpdateData([]byte("updated-data")),   // new semaphore data
    )
    ```

- C++
```

File: ydb/docs/en/core/reference/ydb-sdk/coordination.md [hunk 11]
```diff
@@ -0,0 +1,330 @@
...
+
+    - Go
+
+        ```go
+        err := db.Coordination().CreateNode(ctx,
+            "/path/to/mynode",
+        )
+        ```
+
+    {% endlist %}
```
Comment (asmyasnikov) [inline ydb/docs/en/core/reference/ydb-sdk/coordination.md:23]:
```suggestion
    ```go
    err := db.Coordination().CreateNode(ctx,
        "/path/to/mynode",
    )
    ```
```

**Review by fomichev3000 — APPROVED** [review]

**Pull request merged**